src/train.py

```diff
@@ -101,18 +131,48 @@ def main(argv):
             labels = [CLASS_INDEX[label] for label in labels]
             labels = torch.tensor(labels).to(DEVICE)
 
-            # Zero the parameter gradients
-            optimizer.zero_grad()
+            style_vec = torch.rand((FLAGS.batch, 100), device=DEVICE)
+            char_vec = char_vector(FLAGS.batch, DEVICE)
+
+            # add index for fake character
+            fake_vec = 71 * torch.ones(FLAGS.batch, 1).to(DEVICE)
```

Comment:
The Characters array is in this file we shouldnt' have 71's strewn around

Reply:
huh?

Reply:
There's an array of the characters we care about. we can do len(CHARACTERS) + 1 rather than having a bunch of hard coded 71s